dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -117,25 +91,6 @@ def test_forward_ip():
         assert cluster.scheduler.ip == default_ip
 
 
-@pytest.mark.parametrize("Cluster", [])
```

Comment:
Actually I did not know what to make out of those. There are a few later that probably should also be addressed. Eg. also by moving to the apropriate cluster specific testing...